statchatexpress/app.js

```diff
@@ -70,13 +89,47 @@ app.post("/users/login", async (req, res) => {
     if (user.Password !== Password) {
       return res.status(400).json({ message: "Incorrect password" });
     }
-
-    res.json({ message: "Login successful" });
+    req.session.userId = user.dataValues.id;
+    req.session.save(function () {
+      res.json({ message: "Login successful" });
+    });
   } catch (err) {
     res.status(500).json({ message: err.message });
   }
 });
 
+app.post("/users/profile", async (req, res) => {
+  try {
+    const { gamertag, platform, gameName } = req.body;
+    const userId = req.session.userId;
+
+    const user = await User.findByPk(userId);
+
+    if (user) {
+      let userGame = await UserGame.findOne({
+        where: { UserId: userId },
```

Comment:
When you add additional APIs you're also going to need to pass in the gameName here.

Reply:
i will make note of this in my future 2nd api implementation